packages/web/package.json

```diff
@@ -28,7 +28,7 @@
     "@tailwindcss/aspect-ratio": "^0.2.0",
     "@tailwindcss/forms": "^0.2.1",
     "@tailwindcss/typography": "^0.3.1",
-    "@tydoc/extractor": "workspace:packages/extractor",
+    "@tydoc/extractor": "workspace:*",
```

Comment:
What does this do?

Reply:
It knows the package is named "@tydoc/extractor" so there is no need to tell it where it is as yarn already knows